src/content/docs/integrate/connected-apps/patreon-connected-app.mdx

```diff
@@ -12,7 +12,7 @@ app_context:
 
 You can connect Patreon to your Kinde account so you can manage Patreon members and campaigns.
 
-Before you can add Patreon as a connected app in Kinde, you need to [set up a machine to machine application](/developer-tools/kinde-api/add-a-m2m-application-for-api-access/) to connect to Kinde’s API.
+Before you can add Patreon as a connected app in Kinde, you need to [set up a machine to machine application](/developer-tools/kinde-api/about-m2m-scopes/) to connect to Kinde’s API.
```

Comment:
@clairekinde11 the links in about 20 of these files seem to have been updated to point at `/about-m2m-scopes/` which doesn't seem right. I assume we want them to point at the doc about application setup?

Reply:
Fixed